Portfolio/index.html

```diff
@@ -10,22 +10,22 @@
     <div top="0"></div>
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-    <div style="margin: 10px; padding: 10px">
-        <h1>Develop</h1>
+    <div class="main">

```

Comment:
Consider providing alternative text for the image in line 6 that describes the content or purpose of the image.